paella-matterhorn/ui/watchAbout.shtml

```diff
@@ -132,9 +175,57 @@
               alt="The playback speed control" />
             </p>
           </div>
+          <div class="help_section all" id="change_resolution">
+            <h2 >Change video resolution <span class="vod">/ or view just the presentation video</span></h2>
+            <p>You can change the video resolution via the resolutions menu.
+              Click it and you'll get a set of choices. </p>
+            <p class="vod">Multiple videos may show MULTISTREAM resolution options, or MULTISTREAM and SINGLESTREAM.
+              When MULTISTREAM and SINGLESTREAM both appear, the SINGLESTREAM options are for playing only the presentation video.
```

Comment:
James request to get Tom's team input on this wording to get it more generalized.